storybook/src/docs/components/IconGalleryCategories.tsx

```diff
@@ -49,21 +49,45 @@ export const MediaIcons = [
   'PreviousIcon',
   'ReplayIcon',
   'RewindIcon',
-  'VolumeOffIcon',
   'VolumeOffFillIcon',
-  'VolumeOnIcon',
+  'VolumeOffIcon',
   'VolumeOnFillIcon',
+  'VolumeOnIcon',
 ]
 
 export const SeverityIcons = [
-  'ErrorIcon',
   'ErrorFillIcon',
-  'InfoIcon',
+  'ErrorIcon',
   'InfoFillIcon',
-  'SuccessIcon',
+  'InfoIcon',
   'SuccessFillIcon',
-  'WarningIcon',
+  'SuccessIcon',
   'WarningFillIcon',
+  'WarningIcon',
+]
+
+export const UniversalIcons = [
+  'CloseIcon',
+  'MaximizeIcon',
+  'MinimizeIcon',
+  'SaveFillIcon',
+  'SaveIcon',
+  'SearchIcon',
+  'VotingBallotIcon',
+]
+
+export const FormattingIcons = [
```

Comment:
We could include Table and Image here and rename the group to ‘Text Editor’, or is that too much? Would we miss many other options for a toolbar?

I think we should redesign this page anyway, so fine with me to leave as is.

Reply:
A toolbar can be many things, i would advise against this change.